spec/requests/api/v1/npq_profiles_spec.rb

```diff
@@ -194,52 +194,51 @@ def owner
       let(:json) { json_hash.to_json }
 
       it "creates the npq validation data" do
-        Timecop.freeze(Date.new(2023, 3, 20)) do
```

Comment:
Why was this removed? I think it was just added as a part of the cohorts changes.